src/pages/overview/document-generation-api/stylingformattingtags.md

```diff
@@ -86,7 +88,7 @@ JSON representation of the input data:
 
 - <u\> (underline)
 
-- <p\> (para break)
+- <p\> (para)
```

Comment:
When people use self closing tags, they do not write it this way, they use <p/>. ALso, underline can't be self closing.

Reply:
This is not a self-closing tag. This is the markdown syntax to escape HTML Tags and show them as it is.
In the final documentation it will shown as `<p>`.